solr/core/src/test/org/apache/solr/update/CustomTLogDirTest.java

```diff
@@ -36,8 +36,15 @@ public class CustomTLogDirTest extends SolrTestCaseJ4 {
       new EmbeddedSolrServerTestRule() {
         @Override
         protected void before() {
+          System.setProperty("solr.directoryFactory", "solr.NRTCachingDirectoryFactory");
           solrClientTestRule.startSolr(LuceneTestCase.createTempDir());
         }
+
+        @Override
```

Comment:
STCJ4 arranges for props to be cleared

Reply:
thanks, removed this in 9dedb54121bb9822d8eeb2f301f47a2e09556cdd